.gitlab/functional_test/regression_detector.yml

```diff
@@ -1,5 +1,6 @@
 single-machine-performance-regression_detector:
   stage: functional_test
+  timeout: 1h10
```

Comment:
Given SMP's error p50 and p99 are both 2h01m, I'd expect a large fraction of those errors -- which I'd suspect are timeouts -- are due to checking SMP's ECR for an image to use in Regression Detector performance comparisons (see https://github.com/DataDog/datadog-agent/blob/231be1816118dabc108a17bb07339f15c2b42189/.gitlab/functional_test/regression_detector.yml#L61). https://github.com/DataDog/datadog-agent/pull/29679 should help address that failure mode, so I think a 1h10m timeout could be good. If I wanted to look at the distribution of `single-machine-performance-regression_detector` job durations, what query would I use in Datadog CI Visibility? It would be useful for Single Machine Performance to be able to look at this information as part of operations monitoring.

Reply:
I think I figured out which query to use. Thanks for looking into these CI duration issues!